src/App.js

```diff
@@ -92,13 +92,18 @@ class App extends Component {
   }
 
   render() {
-    return (
-      <div>
-        <Header user={this.state.user} />
-        <img src={logo} alt="logo" />
-        <Footer />
-      </div>
-    );
+    const isLoggedIn = !!this.state.user;
+    if (isLoggedIn) {
+      return (
+        <div>
+          <Header />
+          <img src={logo} alt="logo" />
+          <Footer />
+        </div>
+      );
+    } else {
+      return <div>Loading...</div>;
```

Comment:
Will this change in a future? If it won't, we should use the react-intl lib here.

Reply:
Yes will change. Need Mariana define something..

Reply:
By the moment, I think that `Loading...` is enough acceptable, or you could choose another option, like a spinner. 

So, define an option for ourselves, @MarianaMS will have the opportunity of change it in the future. In the meantime let's make it polished and following all of our requirements, for example using i18n.

Quiero decir, aunque algo no esté definido, no lo haganos a medias. Tomemos una decisión, aunque sea simple y hagamosla bien pulida.